lib/ddtrace/transport/http/adapters/unix_socket.rb

```diff
@@ -8,15 +9,14 @@ module HTTP
       module Adapters
         # Adapter for Unix sockets
         class UnixSocket < Adapters::Net
-          DEFAULT_TIMEOUT = 1
-
           attr_reader \
             :filepath,
             :timeout
 
-          def initialize(filepath, options = {})
-            @filepath = filepath
-            @timeout = options.fetch(:timeout, DEFAULT_TIMEOUT)
+          # @deprecated Positional parameters are deprecated. Use named parameters instead.
+          def initialize(filepath = nil, **options)
+            @filepath = filepath || options[:filepath]
```

Comment:
It just occurred to me that calling it "filepath" is not part of the API, since this was a positional argument. Thus, I think we can just call it `uds_path` everywhere (including the docs), because the previous API never actually relied on that name.

Reply:
I had to keep `attr_reader :filepath`, as to make super sure this is not breaking, but otherwise a full rename has been made!